server_release_template/web_gui/js/app/nls/settings.js

```diff
@@ -1,98 +1,173 @@
+        "_root_headset_controllers_content_positionOffsetLeft_0.name": "X",
+        "_root_headset_controllers_content_positionOffsetLeft_1.name": "Y",
+        "_root_headset_controllers_content_positionOffsetLeft_2.name": "Z",
         "_root_headset_controllers_content_rotationOffsetLeft.name": "Rotation offset",
         "_root_headset_controllers_content_rotationOffsetLeft.description": "Rotation offset in degrees for the left controller.\nFor the right controller, rotations along the Y and Z axes are mirrored",
-        "_root_headset_controllers_content_rotationOffsetLeft_0.name": "x",
-        "_root_headset_controllers_content_rotationOffsetLeft_1.name": "y",
-        "_root_headset_controllers_content_rotationOffsetLeft_2.name": "z",
+        "_root_headset_controllers_content_rotationOffsetLeft_0.name": "X",
+        "_root_headset_controllers_content_rotationOffsetLeft_1.name": "Y",
+        "_root_headset_controllers_content_rotationOffsetLeft_2.name": "Z",
         "_root_headset_controllers_content_hapticsIntensity.name": "Haptics intensity",
-        "_root_headset_headsetEmulationMode.name": "Headset emulation mode",
-        "_root_headset_headsetEmulationMode.description": "Emulates different headsets for better compatibility",
-        "_root_headset_controllers_content_controllerMode.name": "Controller emulation mode",
-        "_root_headset_controllers_content_controllerMode.description": "Emulates different controller for better compatibility or enables hand tracking",
+        "_root_headset_controllers_content_hapticsIntensity.description": "Factor to reduce or increase the intensity of the vibration of the controls.",
         "_root_headset_trackingSpace-choice-.name": "Tracking Space",
         "_root_headset_trackingSpace-choice-.description": "Sets what the headset uses as its reference for tracking and how the center of the space is defined. Stage tracking space behaves like a wired headset: the center of the space stays in one place after recentering the headset.",
-        "_root_headset_trackingSpace_local-choice-.name": "Local (Headset centered)",
-        "_root_headset_trackingSpace_stage-choice-.name": "Stage (Room centered)",
+        "_root_headset_trackingSpace_Local-choice-.name": "Local (Headset centered)",
+        "_root_headset_trackingSpace_Stage-choice-.name": "Stage (Room centered)",
+        // Connection tab
+        "_root_connection_tab.name": "Connection",
+        "_root_connection_disableThrottling.name": "Disable throttling",
+        "_root_connection_disableThrottling.description": "Disables the throttling. Send data as fast as possible",
+        "_root_connection_bufferOffset.name": "Buffer offset",
+        "_root_connection_bufferOffset.description": "Offset to increase or decrease the calculated client buffer size. The client buffer can not be negative",
+        "_root_connection_suppressFrameDrop.name": "Suppress frame drops",
```

Comment:
Some settings like suppressFrameDrop and frameQueueSize were deleted (because they were actually unused). Please review all settings and remove the ones not present as of v14.0.0.

Reply:
Done I also added comment about which parameter are for advanced option only.